pkg/kudoctl/cmd/package_create.go

```diff
@@ -54,6 +59,8 @@ func newPackageCreateCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.StringVarP(&pkg.destination, "destination", "d", ".", "Location to write the package.")
 	f.BoolVarP(&pkg.overwrite, "overwrite", "w", false, "Overwrite existing package.")
+	f.StringVarP((*string)(&pkg.output), "output", "o", "", "Output format")
```

Comment:
This is `kudo package create -o yaml`, right? I think this might be misinterpreted as the output format of the package itself. At least the flag help should make it clear.

Reply:
Good point. Adjusted that.